src/quo2/components/counter/step/__tests__/step_component_spec.cljs

```diff
@@ -0,0 +1,19 @@
+(ns quo2.components.counter.step.--tests--.step-component-spec
```

Comment:
oh actually I think we prefer the name space is
`quo2.components.counter.step.component-spec` because we already have the separated folder for `step`